src/components/screens/home/balance-card.tsx

```diff
@@ -62,10 +70,13 @@ const BalanceCard = () => {
     <>
       <Card>
         <CardContent>
-          <h3 style={{ marginTop: 6 }}>{strings.timebank.balance}</h3>
+          <h3 style={{ marginTop: 6 }}>{`${strings.timebank.balance}`}</h3>
```

Comment:
```suggestion
          <h3 style={{ marginTop: 6 }}>{strings.timebank.balance}</h3>
```
No need to use string interpolation there.